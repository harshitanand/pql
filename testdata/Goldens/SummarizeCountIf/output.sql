WITH "subquery0" AS (SELECT "Directory" AS "Directory", sum(CASE WHEN coalesce(NOT endsWith("FileName", '_test.go'), FALSE) THEN 1 ELSE 0 END) AS "NonTestFiles" FROM "SourceFiles" GROUP BY "Directory")
SELECT * FROM "subquery0" ORDER BY "Directory" ASC NULLS FIRST;
