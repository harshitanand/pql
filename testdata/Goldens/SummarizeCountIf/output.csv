.,1
parser,4
